pkg/client/unversioned/remotecommand/resize.go

```diff
@@ -0,0 +1,55 @@
+/*
+Copyright 2017 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package remotecommand
+
+import (
+	"encoding/json"
+	"io"
+
+	"k8s.io/apimachinery/pkg/util/runtime"
+)
+
+// Size represents the width and height of a terminal.
+type Size struct {
```

Comment:
@sttts are you ok with moving definition of Size and TerminalSizeQueue to client-go/tool/remotecommand? We did the move to break the dependency on pkg/util/terms.